src/database/queries/userExtension.js

```diff
@@ -1,4 +1,5 @@
 const MenteeExtension = require('../models/index').UserExtension
```

Comment:
@nevil-mathew please change the path to @database/models/index

Reply:
Existing code, updated.